pro/initialize.go

```diff
@@ -1,33 +1,33 @@
 //go:build ee
```

Comment:
may need to change this to pro as well since the build tags are changing

Reply:
oh.. i now see the revert. might also want to check changes in .github/workflows/test.yml

Reply:
thx!